x-pack/plugins/ml/common/util/object_utils.ts

```diff
@@ -34,3 +34,14 @@ export const isPopulatedObject = <U extends string = string>(
       requiredAttributes.every((d) => ({}.hasOwnProperty.call(arg, d))))
   );
 };
+
+/**
+ * Get the first key in the object
+ * getFirstKeyInObject({ firstKey: {}, secondKey: {}}) -> firstKey
+ */
+export const getFirstKeyInObject = (arg: unknown): string | undefined => {
+  if (isPopulatedObject(arg)) {
+    const keys = Object.keys(arg);
```

Comment:
Since JS doesn't guarantee the order of keys, should we sort the keys alphabetically to make sure the function returns 100% consistent results?